airbyte-integrations/connectors/source-monday/integration_tests/expected_records.jsonl

```diff
@@ -69,7 +69,8 @@
-{"stream": "updates", "data": {"assets": [], "body": "<p>\ufeffTest</p>", "created_at": "2022-11-21T14:41:21Z", "creator_id": "36694549", "id": "1825302913", "item_id": "3555437747", "replies": [{"id": "1825303266", "creator_id": "36694549", "created_at": "2022-11-21T14:41:29Z", "text_body": "\ufeffTest test", "updated_at": "2022-11-21T14:41:29Z", "body": "<p>\ufeffTest test</p>"}], "text_body": "\ufeffTest", "updated_at": "2022-11-21T14:41:29Z"}, "emitted_at": 1670509894939}
-{"stream": "updates", "data": {"assets": [], "body": "<p>\ufeffHey there \ud83d\udc4b</p><p>\ufeffThis is an update, we usually use this to...update \ud83d\ude04</p><p>\ufeffWe love to communicate with the context of a specific item.</p><p>\ufeff</p><p>\ufeffRight above this box, there are tabs for different item views which can also be used for apps.</p>", "created_at": "2022-06-08T12:53:39Z", "creator_id": "-7", "id": "1825206780", "item_id": "3555179351", "replies": [], "text_body": "\ufeffHey there \ud83d\udc4b\n\n\ufeffThis is an update, we usually use this to...update \ud83d\ude04\n\n\ufeffWe love to communicate with the context of a specific item.\n\n\ufeff\n\n\ufeffRight above this box, there are tabs for different item views which can also be used for apps.", "updated_at": "2022-11-21T14:04:40Z"}, "emitted_at": 1670509894942}
+{"stream": "updates", "data": {"assets": [], "body": "<p>\ufeffTest</p>", "created_at": "2022-11-21T14:41:21Z", "creator_id": "36694549", "id": "1825302913", "item_id": "3555437747", "replies": [{"id": "1825303266", "creator_id": "36694549", "created_at": "2022-11-21T14:41:29Z", "text_body": "Test test", "updated_at": "2022-11-21T14:41:29Z", "body": "<p>\ufeffTest test</p>"}], "text_body": "Test", "updated_at": "2022-11-21T14:41:29Z"}, "emitted_at": 1676683112518}
```

Comment:
Ran some tests around this, some of the emoji codes don't render right in `text_body`. Not sure if this was a change on monday's side. And there's a third record in the `updates` stream when I tested. Retesting this passed the tests